FirebaseOAuthUI/Sources/FUIOAuth.m

```diff
@@ -196,11 +213,19 @@ + (FUIOAuth *)yahooAuthProvider {
 }
 
 + (FUIOAuth *)appleAuthProvider {
+    return [FUIOAuth appleAuthProviderWithAuthUI:[FUIAuth defaultAuthUI]];
+}
+
++ (FUIOAuth *)appleAuthProviderWithAuthUI:(FUIAuth *)authUI {
   UIUserInterfaceStyle style = UITraitCollection.currentTraitCollection.userInterfaceStyle;
-  return [self appleAuthProviderWithUserInterfaceStyle:style];
+  return [self appleAuthProviderWithAuthUI:authUI userInterfaceStyle:style];
 }
 
 + (FUIOAuth *)appleAuthProviderWithUserInterfaceStyle:(UIUserInterfaceStyle)userInterfaceStyle {
+    return [FUIOAuth appleAuthProviderWithAuthUI:[FUIAuth defaultAuthUI] userInterfaceStyle:userInterfaceStyle];
+}
+
++ (FUIOAuth *)appleAuthProviderWithAuthUI:(FUIAuth *)authUI userInterfaceStyle:(UIUserInterfaceStyle)userInterfaceStyle {
```

Comment:
nit: Line break after the first parameter

Reply:
Fixed